src/parser/jobs/sam/modules/ThirdEye.js

```diff
@@ -71,14 +76,26 @@ export default class ThirdEye extends Module {
 		}))
 
 		this.suggestions.add(new TieredSuggestion({
-			icon: ACTIONS.MERCIFUL_EYES.icon,
-			content: <Trans id="sam.thirdeye.suggestion.unspenteyes.content"> Never let a proc go to waste, spend your procs from <ActionLink {...ACTIONS.THIRD_EYE}/> on either <ActionLink {...ACTIONS.HISSATSU_SEIGAN}/> for a small dps gain or using <ActionLink {...ACTIONS.MERCIFUL_EYES}/> for a tiny amount of healing and a 20% emnity reduction  </Trans>,
+			icon: ACTIONS.HISSATSU_SEIGAN.icon,
+			content: <Trans id="sam.thirdeye.suggestion.unspenteyes.content"> Never let a proc go to waste, spend your procs from <ActionLink {...ACTIONS.THIRD_EYE}/> on <ActionLink {...ACTIONS.HISSATSU_SEIGAN}/> for a small dps gain.  </Trans>,
 			tiers: {
 				1: SEVERITY.MINOR,
 				10: SEVERITY.MEDIUM,
 			},
 			value: unspentEyes,
 			why: <Trans id ="sam.thirdeye.suggestion.unspenteyes.why"> {unspentEyes} of your procs were not spent. </Trans>,
 		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.MERCIFUL_EYES.icon,
+			content: <Trans id= "sam.thirdeye.suggestion.badeyes.content"> As of 5.0, <ActionLink {...ACTIONS.MERCIFUL_EYES}/> no longer reduces enmity, you should never use it now. </Trans>,
```

Comment:
No need to compare to previous xpac. Something like "The heal provided by &lt;Merciful Eyes> is not worth the loss of damage from &lt;Seigan>. If a self heal is required, consider &lt;Second Wind>" (or summat).